src/bin/command.rs

```diff
@@ -214,6 +235,24 @@ impl Command {
                 let tx = wallet.stake(addr, amt, gas).await?;
                 Ok(RunResult::Tx(tx.hash()))
             }
+            Command::StakeAllow {
+                addr,
+                bls,
+                gas_limit,
+                gas_price,
+            } => {
+                let addr = match addr {
+                    Some(addr) => wallet.claim_as_address(addr)?,
+                    None => wallet.default_address(),
+                };
+                let mut gas = Gas::new();
```

Comment:
Nit: add a new line between line 247 and line 248.

Reply:
This one is automatically  removed by `cargo fmt`